app/build.gradle

```diff
@@ -83,30 +83,34 @@ android {
             enableSplit = false
         }
     }
+    kotlinOptions {
+        freeCompilerArgs += "-opt-in=kotlin.RequiresOptIn"
```

Comment:
Kotlin 1.6 is way stricter with how experimental annotations are handled, so this is now necessary.